strongbox-data-service/src/main/resources/etc/conf/janusgraph_EMBEDDED.yaml

```diff
@@ -0,0 +1,7 @@
+strongbox.db:
+    janus-graph:
+        storage-username: cassandra
+        storage-password: cassandra
+        storage-host: 127.0.0.1
+        storage-port: 49142
+        storage-root: ${strongbox.vault}/db
```

Comment:
Could all of these be `storage.xyz`, please? Thanks!

Reply:
this is the naming convention that Spring using to map it on property names. So `storage-username` will be mapped to `JanusGraphProperties.storageUsername`.